conf/timeseries-dev-source.conf

```diff
@@ -19,6 +19,11 @@
     #   keyspace = "filodb_prom"
     #   downsample-keyspace = "filodb_prom_downsample"
     # }
+   # Default Kafka host for local testing
+    kafkaHost=localhost
+    kafkaPort=9092
+    kafkaPort=${?KAFKA_PORT}
+    kafkaBootstrapServers = [ ${?kafkaHost}":"${?kafkaPort} ]
```

Comment:
The bulk of the configs in this file are kebab-case. Could these be updated to match?